CoffeeChat/View/SelfSizingCollectionView.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SelfSizingCollectionView.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/18/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+/// SelfSizingCollectionView is a collection view that calculates its own height and does not require height constraints
+class SelfSizingCollectionView: UICollectionView {
+    override func reloadData() {
+        super.reloadData()
+        self.invalidateIntrinsicContentSize()
+    }
+
+    override var intrinsicContentSize: CGSize {
```

Comment:
Can the overriden variable come before the function? add place after class too.

Reply:
Agreed on moving the variable up